my.css

```diff
@@ -0,0 +1,168 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header, .collection, {
+	z-index: 2;
+	position: relative;
+	color: white;
+}
+.wraper::before {
+	content: '';
+	position: absolute;
+	left: 0;
+	top: 0;
+	width: 400px;
+	height: 900px;
+	background-color: black;
+	z-index: -1;
+}
+.burgermenu {
+  position: relative;
+  float: right;
+  top: -75px;
+  right: 0;
+  width: 92px;
+  height: 92px;
+  background: lightgray;
+}
+
+.burger {
+  position: absolute;
+  width: 34.7px;
+  height: 5px;
+  margin-left: 27px;
+  margin-top: 31px;
+  background: #000;
+}
+
+.burger::before {
+  content: "";
+  position: absolute;
+  width: 34.7px;
+  height: 5px;
+  margin-left: 1px;
+  margin-top: 16px;
+  background: #000;
+}
+
+.header__promo {
+  width: 29%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 25px;
+
+}
+
+.logo__promo {
+  color: #fff;
+  padding-top: 40px;
+  padding-left: 40px;
+}
+
+.ru__link {
+  display: inline-block;
+  color: #fff;
+  padding-top: 40px;
+  padding-left: 10px;
+  text-decoration: none;
+}
+
+.collection {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 750px;
+  z-index: 1;
+
+}
+
+.newcollection {
+  display: flex;
+  flex-direction: column;
+  margin: 30px;
+  width: 40%;
+  height: 600px;
+  padding: 20px;
+  
+}
+
+.newcollection__title {
+  margin: 0; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)